### PR TITLE
Get sequence from SequenceStore if not found locally

### PR DESCRIPTION
**Pull request by patuwwy**
**What?**  
Get sequence from Platform when Sequence not found locally.

**Review checks:**

These aspects need to be checked by the reviewer:

- [ ] Verify and confirm operation (please post a screenshot) 
- [ ] All STH tests pass
- [ ] All [Scramjet Cloud Platform](https://docs.scramjet.org/platform) tests pass
- [ ] Documentation is updated or no changes

### CODE REVIEW

**Comment by patuwwy** [comment]
Requires https://github.com/scramjetorg/transform-hub/pull/643. Contains that branch

**Comment by alicja-gruzdz** [comment]
Working great! :clap: 
1. Paired STH with the Manager 2. Checked the STH load and logs 3. Ran the Sequence through the Manager.
Screenshot from 2022-08-17 11-40-46

**Comment by BKscramjet** [comment]
Task linked: [SHN-691 Implement Sequence Store Adapter Class](https://app.clickup.com/t/24308805/SHN-691)

File: packages/adapters/src/index.ts [hunk 1]
```diff
@@ -10,3 +10,6 @@ export * from "./docker-networking";
 
 export * from "./kubernetes-sequence-adapter";
 export * from "./kubernetes-instance-adapter";
+
+export * from "./process-sequence-adapter";
```
Comment (MichalCz) [inline packages/adapters/src/index.ts:14]: We should consider some tree-shaking here at some point. k8s adapter is making STH quite heavy.

**Review by MichalCz — APPROVED** [review]
Needs verification, but code's alright

**Review by alicja-gruzdz — APPROVED** [review]
Verify STH+MM was done in the CPM repo.
The verify of only STH + S3 is best to do after replacing pico with minio. Please correct me if I'm wrong. This replacement could be done later on.

**Comment by alicja-gruzdz** [comment]
Here we go! :crossed_fingers:

**Comment by a-tylenda** [comment]
> 

🤞🏼

**Pull request merged**